src/renderer/i18n/locales/it-IT.js

```diff
@@ -0,0 +1,964 @@
+    ERROR: {
+      LESS_THAN_MINIMUM: 'L\'importo minimo è {amount}',
+      NOT_ENOUGH_AMOUNT: 'L\'importo massimo è {amount}',
+      NOT_VALID: 'L\'importo non è valido',
+      REQUIRED: 'L\'importo è necessario'
+    },
+    LABEL: 'Importo'
+  },
+
+  INPUT_DELEGATE: {
+    ERROR: {
+      REQUIRED: 'Il nome del delegato, l\'indirizzo o la chiave pubblica sono obbligatori',
+      USERNAME_NOT_FOUND: 'Non è stato possibile trovare il delegato \'{0}\'',
+      ADDRESS_NOT_FOUND: 'Non è stato possibile trovare il delegato con l\'indirizzo \'{0}\'',
+      PUBLIC_KEY_NOT_FOUND: 'Non è stato possibile trovare il delegato con la chiave pubblica \'{0}\''
+    },
+    SEARCH_HINT: 'Puoi cercare per nome utente, indirizzo o chiave pubblica'
+  },
+
+  INPUT_FEE: {
+    MINIMUM: 'Minimo',
+    AVERAGE: 'Media',
+    MAXIMUM: 'Massimo',
+    INPUT: 'Immetti',
+    ADVANCED: 'Avanzato',
+    LOW_FEE_NOTICE: 'Le transazioni con commissioni basse potrebbero non essere mai confermate',
+    ADVANCED_NOTICE: 'Fai attenzione a quale commissione andrai a scegliere, perché se dovesse essere troppo alta, costerà più del necessario',
+    UNIQUE: 'La commissione di rete è stata impostata sul valore statico di {fee}',
+    ERROR: {
+      NOT_VALID: 'La commissione non è valida',
+      LESS_THAN_MINIMUM: 'La commissione minima è {fee}',
+      MORE_THAN_MAXIMUM: 'La commissione massima è {fee}'
+    }
+  },
+
+  INPUT_GRID: {
+    MORE: 'Mostra di più'
+  },
+
+  INPUT_GRID_MODAL: {
+    TITLE: 'Seleziona'
+  },
+
+  WALLET_SELECTION: {
+    PROFILE: 'Profilo Mittente',
+    WALLET: 'Portafoglio Mittente'
+  },
+
+  MODAL_ADDITIONAL_LEDGERS: {
+    CANCEL: 'Annulla',
+    INFO: 'Scegli la quantità massima di portafogli Ledger che desideri mostrare. Attualmente stai mostrando {quantity} portafogli.',
+    LARGE_QUANTITY: 'Stai provando a caricare un numero elevato di portafogli. Ciò potrebbe causare un caricamento lento o il timeout del dispositivo.',
+    LOAD: 'Carica Portafogli',
+    QUANTITY: 'Quantità',
+    TITLE: 'Carica portafogli Ledger aggiuntivi'
+  },
+
+  MODAL_CONFIRMATION: {
+    CANCEL: 'No, annulla',
+    CONTINUE: 'Sì, sono sicuro',
+    TITLE: 'Conferma'
+  },
+
+  MODAL_QR_CODE: {
+    TITLE: 'Codice QR',
+    SUBTITLE: 'Scannerizza per l\'Indirizzo'
+  },
+
+  MODAL_QR_SCANNER: {
+    ERROR: {
+      NOT_ALLOWED: 'Dobbiamo accedere alla webcam per scannerizzare il codice QR',
+      NOT_FOUND: 'Non siamo riusciti a trovare una webcam',
+      NOT_SUPPORTED: 'La pagina non viene servita su una connessione sicura (https)',
+      NOT_READABLE: 'Non siamo in grado di leggere lo stream della webcam, forse è già in uso?',
+      OVERCONSTRAINED: 'Sembra che non siamo riusciti a trovare una webcam corretta.',
+      STREAM: 'Sembra che tu stia utilizzando un browser non supportato. Questo è molto curioso ..'
+    },
+    LOADING: 'Loading webcam',
+    INSTRUCTION: 'Pisiziona il codice QR davanti alla webcam',
+    TITLE: 'QR Scanner',

```

Comment:
Is this correct?

Reply:
Better to reverse it. The word "scanner" is ok.

TITLE: 'Scanner QR'

Reply:
There is a typo:

INSTRUCTION: 'P**o**siziona il codice QR davanti alla webcam'